tools/codegen/core/gen_experiments.py

```diff
@@ -143,12 +143,15 @@ def _GenerateExperimentFiles(args, mode):
         _EXPERIMENTS_SRC_FILE, _EXPERIMENTS_HDR_FILE, mode
     )
 
+    print("Generating experiments.bzl")
+    if mode == "test":
+        compiler.GenExperimentsBzl(mode, "bazel/test_experiments.bzl")
+    else:
+        compiler.GenExperimentsBzl(mode, "bazel/experiments.bzl")
+
     if mode == "test":
         print("Generating experiments tests")
```

Comment:
This can be moved into the if-block above.

Reply:
Fixed.